developer-experience/recipes/cross-platform-tasks.md

```diff
@@ -0,0 +1,64 @@
+# Cross Platform Tasks
+
+There are several options to alleviate cross-platform compabilitity issues.
+
+- Docker or container based - using containers as development machines allows developers to get started with minial setup and and abstrats the development environment from the host OS by having it run in a container.
+
+  - Running VS Code in a Docker container and using [Dev Containers](https://code.visualstudio.com/docs/remote/containers) can also help in standardizing the local developer experience across the team.
+  - Follow a tutorial on [Development in Containers](https://code.visualstudio.com/docs/remote/containers-tutorial)
+  - For samples projects and dev container templates see [VS Code Dev Containers Recipe](https://github.com/microsoft/vscode-dev-containers)
+
```

Comment:
I might suggest adding a reference to our dev containers library - https://github.com/microsoft/code-with-engineering-playbook/tree/master/continuous-integration/devcontainers

Reply:
Great, I will add this in as well! Thank you for your feedback and review!